README.md

```diff
@@ -86,15 +86,15 @@ $ serverless deploy --debug
 The `aws-dynamodb` component requires minimal configuration with built-in sane defaults. Here's a complete reference of the `serverless.yml` file for the `aws-dynamodb` component:
 
 ```yml
-component: aws-dynamodb          # (required) name of the component. In that case, it's aws-dynamodb.
-name: my-table                   # (required) name of your instance.
-org: serverlessinc               # (optional) serverless dashboard org. default is the first org you created during signup.
-app: myApp                       # (optional) serverless dashboard app. default is the same as the name property.
-stage: dev                       # (optional) serverless dashboard stage. default is dev.
+component: aws-dynamodb # (required) name of the component. In that case, it's aws-dynamodb.
+name: my-table # (required) name of your instance.
+org: serverlessinc # (optional) serverless dashboard org. default is the first org you created during signup.
+app: myApp # (optional) serverless dashboard app. default is the same as the name property.
+stage: dev # (optional) serverless dashboard stage. default is dev.
```

Comment:
This is gonna get ugly. Linting sometimes just doesn't make sense.

Reply:
It's prettier not linter that reformatted it.

Anyway I turned off prettification of markdown, as indeed it's not what we want here, and there's no way to enforce Prettier to respect that